admin/test/test_installer.py

```diff
@@ -35,7 +35,7 @@
 POSTGRESQL_USERNAME = 'flocker'
 POSTGRESQL_PASSWORD = 'flocker'
 
-CLOUDFORMATION_TEMPLATE_URL = "https://s3.amazonaws.com/installer.downloads.clusterhq.com/flocker-cluster.cloudformation.json"  # noqa
+CLOUDFORMATION_TEMPLATE_URL = "https://s3.amazonaws.com/installer.downloads.clusterhq.com/flocker-cluster.cloudformation.FLOC-4391.json"  # noqa
```

Comment:
This is temporary. In order to test the new cloudformation script without overriding the existing one.